docs/configuration.md

```diff
@@ -30,6 +30,9 @@
   are both on the same file (count, default 0 (no backpressure))
   e.g. backpressure = 10 [writer = 100.log, slowest reader = 89.log, delta = 11]
 * **hostname** - hostname used in logging/messages (default gethostname())
+* **remove_checkpoint_on_stop** - wether plugin checkpoint should be
```

Comment:
Sorry if I misunderstood but I thought that we should NOT remove the checkpoint if a plugin was abnormally terminated so that one can fix the issue that made the plugin to be terminated and then resume the plugin where it stopped?